core/templates/dev/head/pages/about-page/about-page.directive.html

```diff
@@ -45,7 +45,7 @@ <h2 translate="I18N_ABOUT_PAGE_ABOUT_TAB_HEADING"></h2>
             <p translate="I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_8">
             </p>
 
-            <div class="oppia-static-card-content-wide oppia-about-three-cols-container">
+            <div class="oppia-about-three-cols-container" style="align-content: center">
```

Comment:
Let's not use the inline styling, can you please check whether we can move this to the existing class?

Reply:
I resolved that issue by just removing the inline styling and it works properly.